src/category_theory/groupoid/free_groupoid.lean

```diff
@@ -168,9 +161,9 @@ lemma lift_unique (φ : V ⥤q V') (Φ : free_groupoid V ⥤ V')
 begin
   apply quotient.lift_unique,
   apply paths.lift_unique,
-  apply quiver.symmetrify.lift_unique,
+  fapply @quiver.symmetrify.lift_unique _ _ _ _ _ _ _ _ _,
```

Comment:
Why this regression? Now that you've significantly reduced the diff of this PR, is it still needed?

Reply:
it's either that or a `haveI …` block above the `fapply`. I prefer it like that.

Reply:
But why was it working before and why did it break?

Reply:
Because we have now a class `preserve_reverse` stating that a functor maps reverses to reverse, and `lift_unique` now takes it as an instance argument, but we don't have an instance for the specific function in that case, so I'm just feeding it manually. I think I had a reason to not have it as an instance, but maybe that's not the case. Let me think.

Reply:
Oh, yeah, it's actually a composite:
```
(paths.of ⋙q (quotient.functor red_step ⋙ Φ).to_prefunctor).map_reverse
```
and I don't see it making much sense to have a top level instance just for that. 
Is that bad reasoning?

Reply:
Can't you have instances for
* `map_reverse paths.of`
* `map_reverse (functor.to_prefunctor _)`
* `map_reverse (_ ⋙q _)`

Then the instance will be provided automatically.

Reply:
I think I could, but that would be quite a lot more work. I'd rather do a follow-up PR with it if you don't mind?

Reply:
I see you already have two out of the three instances. So surely the last one is not much harder?